flowdb/sql/04_schema_other.sql

```diff
@@ -78,6 +78,18 @@ CREATE TABLE etl.etl_records (
 	PRIMARY KEY (id)
 );
 
+CREATE TABLE etl.post_etl_queries (
+    cdr_date DATE,
```

Comment:
Can we record the date of the check as well?

Reply:
Good point. 👍

Reply:
I've made it a `TIMESTAMPTZ` rather than a `DATE` column because there is no harm in having the more precise info about when the check was run.